### ISSUE TITLE
Add the ability to open a torrent folder by double-clicking.

### ISSUE THREAD

**Issue by moninahh** [transmission/transmission#7080]
### What is the issue?

Need to add the ability to open a torrent folder by double clicking. Because it is faster and more convenient than pressing "Ctrl+E".

### Which application of Transmission?

Qt app on Windows

### Which version of Transmission?

transmission-4.0.6-qt5-x64

**Comment by killemov** [comment]
Is this the best thing we can think of using double-click? Another good use may be to just execute the payload file. In [Shift](https://github.com/killemov/Shift) double clicking a torrent row opens the files view (or trackers view if the torrent is still a magnet).

**Comment by Pentaphon** [comment]
> Another good use may be to just execute the payload file.

No. That would make it too easy to get people in trouble with suspicious files. I think opening the torrent folder is fine or in the case of single file torrents, opening the download location folder.

**Comment by dftf-stu** [comment]
Or just give the user the choice for what a double-click will do, as other torrent clients currently permit...?

![Image](https://github.com/user-attachments/assets/c20b06ca-a617-48a2-ba14-fb7ea0795c57)

**Comment by elchupanibria** [comment]
Any update on this? This is the only reason I switched to qbittorrent recently, but the behaviour is broken on mac too so I switched back.

**Comment by zeninfinity** [comment]
This is the second time I've looked for a solution for this and would also love this option. 

There was a previous PR with the code already done I think - https://github.com/transmission/transmission/pull/214 - That would allow this functionality.  

Appreciate the hard work on this project.  

Best.
